android/src/main/java/ch/milosz/reactnative/RNZoomUsModule.java

```diff
@@ -355,11 +357,11 @@ public void run() {
           if(paramMap.hasKey("noTextMeetingId") && paramMap.getBoolean("noTextMeetingId")) opts.meeting_views_options = opts.meeting_views_options + view.NO_TEXT_MEETING_ID;
           if(paramMap.hasKey("noTextPassword") && paramMap.getBoolean("noTextPassword")) opts.meeting_views_options = opts.meeting_views_options + view.NO_TEXT_PASSWORD;
 
-          JoinMeetingParams params = new JoinMeetingParams();
+          JoinMeetingParam4WithoutLogin params = new JoinMeetingParam4WithoutLogin();
           params.displayName = paramMap.getString("userName");
           params.meetingNo = paramMap.getString("meetingNumber");
-          if(paramMap.hasKey("password")) params.password = paramMap.getString("password");
-          if(paramMap.hasKey("webinarToken")) params.webinarToken = paramMap.getString("webinarToken");
+          if (paramMap.hasKey("password")) params.password = paramMap.getString("password");
+          if (paramMap.hasKey("zoomAccessToken")) params.zoomAccessToken = paramMap.getString("zoomAccessToken");
```

Comment:
Any reason why removing `webinarToken` for joinMeeting? This was working correctly. Can we have both `zoomAccessToken` and `webinarToken`?

Reply:
no particular reason i just forget it
i double check the docs and fix it in new commit
https://marketplacefront.zoom.us/sdk/meeting/android/us/zoom/sdk/JoinMeetingParam4WithoutLogin.html